src/install/install.zig

```diff
@@ -6296,10 +6348,8 @@ pub const PackageManager = struct {
     var cwd_buf: [bun.MAX_PATH_BYTES]u8 = undefined;
     var package_json_cwd_buf: [bun.MAX_PATH_BYTES]u8 = undefined;
 
-    pub inline fn install(
-        ctx: Command.Context,
-    ) !void {
-        var manager = PackageManager.initMaybeInstall(ctx, null, &install_params, true) catch |err| {
+    pub inline fn install(ctx: Command.Context) !void {
```

Comment:
If you do `bun install react` inside a workspace package does it add to the workspace package's package.json?

Reply:
With this PR it adds it to the root package.

Reply:
`bun install` should install the root workspace package
`bun install react` should:
1) Add `react` as a dependency to the workspace package's package
2) Install the workspace package's dependencies and the root package's dependencies (all workspace package's dependencies is fine for now though since IIRC it works that way currently and we can fix that later)

Reply:
Done in https://github.com/oven-sh/bun/pull/2834/commits/28c24d8b5b412b57f7f00228c98dc46bdede16f7